Tasks/TwineAuthenticateV1/twineauthenticatemain.ts

```diff
@@ -72,6 +139,26 @@ function findDuplicatesInArray<T>(array: Array<T>): Array<T>{
     return array.filter((e, i, a) => a.indexOf(e) !== i);
 }
 
+function removeFromFeedCount(internalFeed: auth.AuthInfo[], externalEndpoints: auth.AuthInfo[], entry: auth.AuthInfo): void {
+    if (internalFeed.includes(entry)) {
+        internalFeed.pop();
```

Comment:
The task was already set up so that it could only authenticate one internal and one external connection max per run, so at most internalFeed and externalEndpoints can just have one entry

Reply:
yes, but the task implements as an array, so if something changed and breaks this assumption that only one feed can be added per run, this function might silently report incorrect telemetry

Reply:
Ok, I ended up changing internalFeed and externalEndpoints to sets because these still have the functionality we need and it makes it easier to remove specific elements